config/sidebar.paper.ts

```diff
@@ -125,6 +125,12 @@ const paper: SidebarsConfig = {
                 "dev/api/event-api/chat-event",
               ],
             },
+            {
+              type: "category",
+              label: "Brigadier Command API",
```

Comment:
Not sure if I like Brig being here. Needs input

Reply:
yeah, something like `Command API (Brigadier)` would be more in line with the sibling categories (sorry cas)

Reply:
That was also my first thought but someone here requested a change so that's what i did...